### DISCUSSION TITLE
[New] Proposition d'ajout de pratique : « Limiter les outils d'analytics et les données collectées »

### DISCUSSION THREAD

**Discussion by rlemaire** [cnumr/best-practices#22] [Bonnes pratiques] [answered]
Le tracking est souvent la première source d'alourdissement des pages pour un bénéfice limité pour l'utilisateur. 

Le fournisseur de service numérique devrait au moins limiter à un outil, qui ne soit pas trop gourmand en ressources.

**Comment by DocRoms** [comment]
Je suis tellement d'avis avec toi là dessus. Il y aurait tellement d'exemples : 
 - éviter d'avoir plusieurs sortes de trackeurs différents
 - éviter de charger des Javascripts important des trackeurs (ça rejoins un peu la règle [BP#19](https://github.com/cnumr/best-practices/blob/main/chapters/BP_019_fr.md) qui dit "Remplacer les boutons officiels de partage des réseaux sociaux"... ce qui évite aussi les trackeurs intégrés.

**Comment by ACTLEM** [comment]
Note équipe projet: Règle de validation: Un seul outil d'analytics + privilégier ceux qui consomment le moins de ressources

**Comment by ACTLEM** [comment, marked as answer]
https://github.com/cnumr/best-practices/issues/137
